service/controller/managementcluster/controller.go

```diff
@@ -78,7 +70,9 @@ func NewController(config ControllerConfig) (*Controller, error) {
 				return new(v1.Service)
 			},
 			Resources: resources,
-			Selector:  selector,
+			Selector: labels.SelectorFromSet(labels.Set{
+				"component": "apiserver",
+			}),
```

Comment:
This is the only change apart from import path changes - `controller.Selector` is no longer a thing and we can just use a labels selector directly.

Reply:
:+1: